doc/releases/changelog-0.25.0.md

```diff
@@ -524,12 +507,37 @@ of operators.
   ({-1: 470, 1: 530}, {-1: 470, 1: 530})
   ```
   
-* An operator called [`qml.FlipSign`](https://pennylane.readthedocs.io/en/latest/code/api/pennylane.FlipSign.html) 
-  is now available.
+* A new experimental return type for QNodes with multiple measurements has been added.
+  [(#2814)](https://github.com/PennyLaneAI/pennylane/pull/2814)
+  [(#2815)](https://github.com/PennyLaneAI/pennylane/pull/2815)
+  [(#2860)](https://github.com/PennyLaneAI/pennylane/pull/2860)
+
+  QNodes returning a list or tuple of different measurements return an intuitive 
+  data structure via `qml.enable_return()`, where the individual measurements are 
+  separated into their own tensors:
+
+  ```python
+  qml.enable_return()
+  dev = qml.device("default.qubit", wires=2)
+  
+  @qml.qnode(dev)
+  def circuit(x):
+      qml.Hadamard(wires=[0])
+      qml.CRX(x, wires=[0, 1])
+      return (qml.probs(wires=[0]), qml.vn_entropy(wires=[0]), qml.probs(wires=0), qml.expval(wires=1))
+  ```
+  ```pycon
+  >>> circuit(0.5)
+  (tensor([0.5, 0.5], requires_grad=True), tensor(0.08014815, requires_grad=True), tensor([0.5, 0.5], requires_grad=True), tensor(0.93879128, requires_grad=True))
+  ```
+
+  In addition, QNodes that utilize this new return type support backpropagation.
+  This new return type can be disabled thereafter via `qml.disable_return()`.
+
+* An operator called `qml.FlipSign` is now available.
   [(#2780)](https://github.com/PennyLaneAI/pennylane/pull/2780)
 
-  Mathematically, [`qml.FlipSign`](https://pennylane.readthedocs.io/en/latest/code/api/pennylane.FlipSign.html)  
-  functions  as follows: 
+  Mathematically, `qml.FlipSign` functions as follows: 
   $\text{FlipSign}(n) \vert m \rangle = (-1)^\delta_{n,m} \vert m \rangle$, where 
   $\vert m \rangle$ is an arbitrary qubit state and $n$ is a qubit configuration:
```

Comment:
Math isn't rendering 😢. How do I do this? 😅

Reply:
```suggestion
  :math:`\text{FlipSign}(n) \vert m \rangle = (-1)^\delta_{n,m} \vert m \rangle`, where 
  :math:`\vert m \rangle` is an arbitrary qubit state and $n$ is a qubit configuration:
```

A weird subtlety of including a markdown document inside of sphinx :(